### PR TITLE
CPATH and LD_LIBRARY_PATH missing defaults for SuiteSparse

### PR DESCRIPTION
**Pull request by ocaisa**
Default values were not included in `CPATH` and `LD_LIBRARY_PATH`

### CODE REVIEW

File: easybuild/easyblocks/s/suitesparse.py [hunk 1]
```diff
@@ -167,8 +167,8 @@ def make_module_req_guess(self):
         """
         guesses = super(EB_SuiteSparse, self).make_module_req_guess()
         guesses.update({
-            'CPATH': [self.config_name],
-            'LD_LIBRARY_PATH': ['UMFPACK/Lib', 'AMD/Lib'],
+            'CPATH': ['include', self.config_name],
+            'LD_LIBRARY_PATH': ['lib', 'UMFPACK/Lib', 'AMD/Lib'],
```
Comment (boegel) [inline easybuild/easyblocks/s/suitesparse.py]: how about changing it to:

```python
guesses['CPATH'].append(self.config_name)

extra_libs = ['AMD/Lib', 'UMFPACK/Lib']
guesses['LD_LIBRARY_PATH'].extend(extra_libs)
guesses['LIBRARY_PATH'].extend(extra_libs)
```

extra bonus points :cookie: if you also tackle the issue reported in #882 while you're at it :-)

**Comment by ocaisa** [comment]
I think #882 might be directly resolved with this change, the problem before (at least for Octave) was that libraries were not being found because the `lib` directory was not in the LD path, if you looked at the config.log you could see it was simply failing to resolve libraries. Adding the additional paths was only a patch for it to resolve libraries since the static libs are still in those paths (I wonder if the static libs they should be moved to `lib` by the easyblock since that is where the so libs are).

**Comment by boegel** [comment]
@ocaisa: see also #863 (mostly unrelated, but it would be nice to tackle the updates all in one go, to avoid having to deal with conflicts)

or, I should get #863 merged first, since the changes in here are fairly trivial

cc @besserox

**Comment by besserox** [comment]
For me #863 is ready to be merged. A minor improvement could be an additional sanity check (cf my last comment in #863) but it can be a separate PR later.

Unfortunately, I won't have any time to look at or test anything in the next 2 weeks.

**Comment by ocaisa** [comment]
I think #863 already fixed this issue...and does a much more comprehensive job, I'm closing.

**Pull request closed without merging**